database/Populator.js

```diff
@@ -30,7 +30,7 @@ const populateRestaurants = () => {
 
 const populateUsers = () => {
   for (let i = 0; i < 100; i++) {
-    let userNames = ['Neon', 'Omega', 'Kookaburra', 'RagingBull', 'Asylum', 'Flurry', 'Ollie', 'Scoobie', 'Brave', 'Guaymas', 'Pinochio', 'Danana', 'Pepita', 'Padila', 'Ninja', 'Bonza'];
+    let userNames = ['Ken Y.', 'Omega S.', 'Johnny J.', 'Bald B.', 'Bison M.', 'Guile T.', 'Ollie C.', 'Scoobie D.', 'Brave H.', 'Rye T.', 'Chun L.', 'Blanka B.', 'Zangief K.', 'Dhalsim B.', 'Sagat P.', 'Vega B.', 'Charlie T.', 'Michael W.', 'Servio L.', 'Trevor P.'];
     let randomIndex = randomNumberGenerator(0, userNames.length);
```

Comment:
Usually, you declared variables and or array outside of the loop so it doesn't create those variable every time it iterates through the for loop

Reply:
Good point